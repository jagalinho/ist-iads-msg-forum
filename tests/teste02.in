A 123 Mensagem de teste
A 123 Outra mensagem de teste
A 567 Ainda outra mensagem de teste
U 123
U 890
O
A 890 A roda do autocarro roda roda
O
X