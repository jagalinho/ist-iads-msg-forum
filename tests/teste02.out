*MESSAGES FROM USER:123
Mensagem de teste
Outra mensagem de teste
*MESSAGES FROM USER:890
*LONGEST SENTENCE:567:Ainda outra mensagem de teste
*LONGEST SENTENCE:567:Ainda outra mensagem de teste
*LONGEST SENTENCE:890:A roda do autocarro roda roda
4
